apps/src/StudioApp.js

```diff
@@ -703,14 +717,16 @@ StudioApp.prototype.showInstructions_ = function(level, autoClose) {
 *  Resizes the blockly workspace.
 */
 StudioApp.prototype.onResize = function() {
-  var visualizationColumn = document.getElementById('visualizationColumn');
-  var gameWidth = visualizationColumn.getBoundingClientRect().width;
-
-  var blocklyDiv = document.getElementById('blockly');
-  var codeWorkspace = document.getElementById('codeWorkspace');
 
-  // resize either blockly or codeWorkspace
-  var div = this.editCode ? codeWorkspace : blocklyDiv;
+  // First, grab the main app container
+  // This is inconsistently named across apps right now
+  // TODO (bbuchanan) : Unify parent app container names
+  var div = document.getElementById('appcontainer');
+  if (!div && this.editCode) {
+    div = document.getElementById('codeWorkspace');
+  } else if (!div && this.isUsingBlockly()) {
+    div = document.getElementById('blockly');
+  }
```

Comment:
I like how you've made it clearer what's happening here. I also like the idea of unifying the name at some point :)